torch/_torch_docs.py

```diff
@@ -8810,6 +8810,33 @@ def merge_dicts(*dicts):
     tensor([ 4,  5,  8])
 """.format(**common_args))
 
+add_docstr(torch.take_along_dim,
+           r"""
+take_along_dim(input, indices, dim, *, out=None) -> Tensor
+
+Returns values from the :attr:`input` at the given 1-d indices along the given dim.
```

Comment:
"Returns" -> "Selects"

Reply:
"at the 1-dimensional indices from :attr:\`indices\` along the given :attr:\`dim\`."